codec/json/doc.go

```diff
@@ -1,3 +1,2 @@
-// Package json contains codec implementations that use Go's standard JSON
-// marshaling.
+// Package json contains a codec that use Go's standard JSON marshaling.
```

Comment:
This feels like it reads a bit awkward. Should it say _uses_?
```suggestion
// Package json contains a codec that uses Go's standard JSON marshaling.
```